jbmc/regression/jbmc/assume-inputs-integral/My.java

```diff
@@ -0,0 +1,84 @@
+class My {
+
+  float floatField;
+  double doubleField;
+
+  static float floatFieldStatic;
+  static double doubleFieldStatic;
+
+  double[] doubleArrField;
+
+  public static void numericalArg(float floatArg, double doubleArg) {
+    if (floatArg == 1.08) {
+      assert false;
+    } else if (doubleArg == 1.08) {
+      assert false;
+    } else if (doubleArg != floatArg) {
```

Comment:
You should test that it doesn't allow Float.NaN, Float.NEGATIVE_INFINITY Float.POSITIVE_INFINITY

Reply:
Added :+1: